qiskit/circuit/library/standard_gates/u2.py

```diff
@@ -49,11 +61,24 @@ class U2Gate(Gate):
 
     **Examples:**
 
+    .. math::
+
+        U2(\phi,\lambda) = RZ(\phi) RY\left(\frac{\pi}{2}\right) RZ(\lambda)
```

Comment:
I think this first equivalence with RZ RY RZ misses a global phase, doesn't it?

Reply:
I hadn't actually checked that one (trusted it to be correct but seems that was an error on my part) but you are correct. $U2(\phi, \lambda) = e^{i\frac{\phi+\lambda}{2}}RZ(\phi)RY(\pi/2)RZ(\lambda)$